src/models/Database.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+
+import { Sequelize } from 'sequelize';
+
+import { StakeRequestRepository } from './StakeRequestRepository';
+
+export default class Database {
+  /* Storage */
+
+  public stakeRequestRepository: StakeRequestRepository;
+
+
+  /* Public Functions */
+
+  /** Creates in memory database. */
+  public static async createInMemory(): Promise<Database> {
```

Comment:
Agree. The `create` method has replaced above-mentioned methods that gets `storage` argument to specify either a database file path or ':memory:' in case of in memory database.